### PR TITLE
generate secrets without exposing the password

### PR DESCRIPTION
**Pull request by ReSearchITEng**
generate secrets without exposing the password as a variable, or risking to leave it as a variable.
Added namespace creation line as well, as it was not created before that step.

### CODE REVIEW

File: install/kubernetes/helm/istio/README.md [hunk 1]
```diff
@@ -73,33 +70,18 @@ $ helm init --service-account tiller
...
-```
-
-Set the namespace where Istio was installed:
-```
-$ NAMESPACE=istio-system
-```
-
-Create secret for Grafana:
-```
-$ cat <<EOF | kubectl apply -f -
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/README.md]: We still need `kubectl apply`

Comment (ReSearchITEng) [reply install/kubernetes/helm/istio/README.md]: sorry, missed that in 2nd secret. Fixed now.

File: install/kubernetes/helm/istio/README.md [hunk 2]
```diff
@@ -58,12 +58,9 @@ $ helm init --service-account tiller
 
 4. If you are enabling `kiali`, you need to create the secret that contains the username and passphrase for `kiali` dashboard:
    ```
-   $ echo -n 'admin' | base64
-   YWRtaW4=
-   $ echo -n '1f2d1e2e67df' | base64
-   MWYyZDFlMmU2N2Rm
-   $ NAMESPACE=istio-system
-   $ cat <<EOF | kubectl apply -f -
+   NAMESPACE=istio-system
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/README.md]: You may want to rebase, as the namespace creation was moved to step 3 at https://github.com/istio/istio/commit/536df4b8cb473b297adfdb3855e427c0ec80a63f

Comment (ReSearchITEng) [reply install/kubernetes/helm/istio/README.md]: conflict resolved

File: install/kubernetes/helm/istio/README.md [hunk 3]
```diff
@@ -64,11 +64,9 @@ $ kubectl create ns $NAMESPACE
 
 5. If you are enabling `kiali`, you need to create the secret that contains the username and passphrase for `kiali` dashboard:
    ```
-   $ echo -n 'admin' | base64
-   YWRtaW4=
-   $ echo -n '1f2d1e2e67df' | base64
-   MWYyZDFlMmU2N2Rm
-   $ cat <<EOF | kubectl apply -f -
+   NAMESPACE=istio-system
+   kubectl create namespace $NAMESPACE || true
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/README.md]: We do not need to create this namespace as it was already created in step 3

**Comment by ReSearchITEng** [comment]
> I signed it!

File: install/kubernetes/helm/istio/README.md (hunk 3 above)
Comment (ReSearchITEng) [reply install/kubernetes/helm/istio/README.md]: I think it came in between. please see now.

File: install/kubernetes/helm/istio/README.md [hunk 4]
```diff
@@ -64,11 +64,8 @@ $ kubectl create ns $NAMESPACE
 
 5. If you are enabling `kiali`, you need to create the secret that contains the username and passphrase for `kiali` dashboard:
    ```
-   $ echo -n 'admin' | base64
-   YWRtaW4=
-   $ echo -n '1f2d1e2e67df' | base64
-   MWYyZDFlMmU2N2Rm
-   $ cat <<EOF | kubectl apply -f -
+   NAMESPACE=istio-system
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/README.md]: Do we still need this? I think the namespace was set in step 3.

File: install/kubernetes/helm/istio/README.md [hunk 5]
```diff
@@ -78,41 +75,34 @@ $ kubectl create ns $NAMESPACE
...
-    app: grafana
-type: Opaque
-data:
-  username: YWRtaW4=
-  passphrase: MWYyZDFlMmU2N2Rm
-EOF
-```
+   Encode username, you can change the username to the name as you want:
+   ```
+   NAMESPACE=istio-system
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/README.md]: Do we still need this? I think the namespace was set in step 3.

**Pull request closed without merging**